sopel/bot.py

```diff
@@ -286,6 +324,11 @@ def msg(self, recipient, text, max_messages=1):
     def say(self, text, recipient, max_messages=1):
         """Send ``text`` as a PRIVMSG to ``recipient``.
```

Comment:
Overlooked this technical-soup one the other day. Sorry about that! Probably because you didn't touch it, and I was focused on the changed/added lines.

```suggestion
        """Send a PRIVMSG to a user or channel.
```